charts/incubator/kopia/questions.yaml

```diff
@@ -18,21 +18,6 @@ questions:
                                     additional_attrs: true
                                     type: dict
                                     attrs:
-                                      - variable: USER
-                                        label: "Kopia User"
-                                        description: "Repository user"
-                                        schema:
-                                          type: string
-                                          default: "secret"
-                                          required: true
-                                      - variable: KOPIA_PASSWORD
-                                        label: "Kopia Password"
-                                        description: "Repository password"
-                                        schema:
-                                          type: string
-                                          default: "secret"
-                                          required: true
-                                          private: true
```

Comment:
Lets not delete these variables
KOPIA_PASSWORD and USER are repo auth env variables -> https://kopia.io/docs/installation/

Reply:
Right